src/test/java/com/google/api/codegen/testdata/ruby_smoke_test_library.baseline

```diff
@@ -0,0 +1,41 @@
+============== file: test/library/v1/library_service_smoke_test.rb ==============
```

Comment:
Having this be generated into the same top level directory as the unit tests will cause problems automating the tests in the rake file. I think it would be better to have this generate something like `smoke_test/library/v1/library_service_smoke_test.rb`.